src/main/resources/db/migration/V001.1__CREATE_BOOKS_TABLE.sql

```diff
@@ -14,4 +14,4 @@ create table books
     isbn             varchar(255),
     primary key (id)
 );
-alter table if exists books add constraint fk_author_id foreign key (author_id) references author;
+alter table books add constraint fk_author_id foreign key (author_id) references author (id);
```

Comment:
I don't think this change here is necessary, and if it is then let's create a new flyway migration script with `V001.2` version